js/templates/userPage/userPageHome.html

```diff
@@ -0,0 +1,81 @@
+<div class="flexRow gutterH">
+  <div class="col4 gutterVMd">
+    <div class="js-userShort">
+      <!-- the userShort template is injected here-->
+    </div>
+    <% if(ob.isModerator) { %>
+    <div class="contentBox padMd2 clrBr clrP clrSh2 js-moderatorInfo">
+      <div class="flexVBot gutterH rowLg snipKids">
+        <div class="tx4 flexExpand"><strong><%= ob.polyT('userPage.moderator') %></strong></div>
+        <div class="clrT2 tx5">
+          <a class="link js-termsLink">
+            <%= ob.polyT('userPage.termsOfService') %>
+          </a>
+        </div>
+      </div>
+      <div class="tx2 clrT2 txCtr rowMd"><%= ob.polyT('userPage.modFee', {modFee: 1.3}) %></div>
+      <div class="flexHCent">
+        <a class="btn clrP clrBr clrSh2 js-addModerator">
+          <span class="js-addModeratorLbl <% if(ob.currentModerator) print('hide') %>">
+            <%= ob.polyT('userPage.addModerator') %>
+          </span>
+          <span class="js-removeModeratorLbl <% if(!ob.currentModerator) print('hide') %>">
+            <%= ob.polyT('userPage.removeModerator') %>
+          </span>
+        </a>
+      </div>
+    </div>
+    <% } %>
+    <div class="informationList listBox clrBr clrP clrSh2">
+      <div class="informationHeader">
+        <div><%= ob.polyT('userPage.information') %></div>
+      </div>
+      <div class="listItem">
+        <div><%= ob.polyT('userPage.website') %></div>
+        <div class="clrT2"><a href="<%= ob.website %>"><%= ob.website %></a></div>
+      </div>
+      <div class="listItem">
+        <div><%= ob.polyT('userPage.email') %></div>
+        <div class="clrT2"><%= ob.email %></div>
+      </div>
+      <div class="listItem">
```

Comment:
Yeah, that's true and if the re-render is not super smooth (i.e. too much state is lost), then we may want to instead just put a message somewhere on the page that says a refresh is needed. My reasoning is that the Save could happen after they've closed the modal and they may be interacting with the page and a forced re-render might be to jarring.

That's what github does:
![image](https://cloud.githubusercontent.com/assets/794517/17608675/eb4b301e-5fe2-11e6-8282-eb017cb0d915.png)